### PR TITLE
add ability to blacklist sharded CFs

### PR DESCRIPTION
**Pull request by lwsanty**
Signed-off-by: lwsanty <lwsanty@gmail.com>

### CODE REVIEW

File: terraform/gen.go [hunk 1]
```diff
@@ -25,10 +25,11 @@ const (
...
 var (
-	fOut    = flag.String("o", "main.tf", "output file")
-	fDot    = flag.String("dot", "../architecture.dot", "output DOT file")
-	fLegacy = flag.Bool("legacy", false, "use legacy functions in deployment")
-	fSeq    = flag.Bool("seq", false, "make the deployment sequential")
+	fOut      = flag.String("o", "main.tf", "output file")
+	fDot      = flag.String("dot", "../architecture.dot", "output DOT file")
+	fLegacy   = flag.Bool("legacy", false, "use legacy functions in deployment")
+	fSeq      = flag.Bool("seq", false, "make the deployment sequential")
+	fFetchers = flag.Bool("fetchers", false, "deploy node fetchers")
```
Comment (dennwc) [inline terraform/gen.go:32]:
```suggestion
	fFetchers = flag.Bool("fetchers", true, "deploy node fetcher CFs")
```
And then pass this flag to the codegen explicitly.

Comment (lwsanty) [reply terraform/gen.go]: Done.

**Comment by lwsanty** [comment]
test

**Comment by lwsanty** [comment]
test

**Pull request merged**